src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -1188,6 +1188,19 @@ static Program()
 CodeStyleOptions.QualifyEventAccess);
         }
 
+        [WorkItem(26893, "https://github.com/dotnet/roslyn/issues/26893")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)]
+        public async Task DoNotReportToQualify_IfInStaticContext7()
+        {
+            await TestMissingAsyncWithOption(
+@"[Obsolete(nameof([|Foo|]))]
+class Program
+{
+    public int Foo { get; set; }
+}",
+CodeStyleOptions.QualifyPropertyAccess);
```

Comment:
can we have a test where the attribute is on something like another field, or potentially on an accessor?  I'm not sure if the check below is sufficient, and i would like to see some additional cases added to see if it is.

Thanks!

Reply:
Thanks Cyrus, very good call. The check passes when an attribute is on a property, but the fixer is still incorrectly offered for an attribute applied to a field.